sway-core/src/parse_tree/literal.rs

```diff
@@ -233,6 +242,9 @@ impl Literal {
             }
             Byte(b) => vec![0, 0, 0, 0, 0, 0, 0, b.to_be_bytes()[0]],
             B256(b) => b.to_vec(),
+            Numeric(_) => {
+                unreachable!("Cannot convert a Numeric Literal with unknown size to bytes")
```

Comment:
Would prefer an `InternalError` to this, although that would be a PITA since this function doesn't return a result yet. Could we default to `u64` in this case?

I know it is technically `unreachable` right now, but we can't guarantee that things won't change (bugs won't be introduced) in the future, and panicking on an `unreachable code` panic is a bad look.

Reply:
Fixed.